templates/company/package.json

```diff
@@ -25,7 +25,8 @@
     "docker:wait-gas": "truffle exec ./node_modules/@aragon/templates-shared/scripts/sleep-until-gaslimit.js --network devnet 6900000",
     "deploy:rpc": "npm run compile && truffle exec ./scripts/deploy.js --network rpc",
     "deploy:devnet": "npm run compile && truffle exec ./scripts/deploy.js --network devnet",
-    "deploy:aragen": "npm run compile && truffle exec ./scripts/deploy.js --network rpc --ens 0x5f6f7e8cc7346a11ca2def8f827b7a0b612c56a1 --dao-factory 0x5d94e3e7aec542ab0f9129b9a7badeb5b3ca0f77 --mini-me-factory 0xd526b7aba39cccf76422835e7fd5327b98ad73c9 --register false",
+    "deploy:harmony": "truffle exec ./scripts/deploy.js --network harmony --ens <ENS Registry address from step 2> --dao-factory <DAO Factory address from step 2>",
```

Comment:
Are `<ENS Registry address from step 2>` and `<DAO Factory address from step 2>` right?

Reply:
We can remove this line, I added it as placeholder to when the mainnet deploy is complete. (Answers same as below)